### PR TITLE
Update app-governance-visibility-insights-view-apps.md

### PR DESCRIPTION
**Pull request by anandd512**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@anandd512 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by dcurwin — APPROVED** [review]

File: CloudAppSecurityDocs/app-governance-visibility-insights-view-apps.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ The app details pane provides additional information on these tabs:
 | Tab name | Description |
 |:-------|:-----|
 | Details | See additional data on the app such as the date first consented and the App ID. To see the properties of the app as registered in Azure AD, select **View app in Azure AD**. |
-| Data usage |See the data usage by the app in the tenant and plot the data usage for SharePoint and Exchange resources. You can fliter usage insights by priority accounts only |
+| Data usage |See the data usage by the app in the tenant and plot the data usage for Exchange, SharePoint, OneDrive and Teams resources. You can fliter usage insights by priority accounts only |
```
Comment (Court72) [inline CloudAppSecurityDocs/app-governance-visibility-insights-view-apps.md:70]:
```suggestion
| Data usage |See the data usage by the app in the tenant and plot the data usage for Exchange, SharePoint, OneDrive, and Teams resources. You can filter usage insights by priority accounts only. |
```

**Comment by dcurwin** [comment]
@Court72 - Why was this PR merged? The author did not sign off yet, and was not ready for the content to be published.

**Comment by anandd512** [comment]
Hi @dcurwin and @Court72,

I thought the tag 'do-not-merge' was sufficient to ensure that this PR is not merged.
Can you please help revert if it has been merged?
Also, please let me know if there's any change I need to do to ensure this does not happen again -
1. If I need to be making changes to the private repo - can you please add me to that repo?
2. If there's any other label that I should add to call out that the PR is not ready to get published?

**Pull request merged**